### PR TITLE
#826 - Verbose output for IT cases

### LINKED ISSUE #826: Make output for IT cases more verbose
Now  only command for execution is logged in integration tests. It would be better to make output of these tests more verbose so that the problem can identified in case of failure.

### PR DESCRIPTION
**Pull request by genryxy**
Closes #826 
Made output for integration tests more verbose by refactoring `TestContainer`

### CODE REVIEW

File: src/test/java/com/artipie/test/TestContainer.java [hunk 1]
```diff
@@ -106,4 +133,20 @@ public void close() {
...
+    private void logStd(final Container.ExecResult exec, final boolean error) {
+        final String outputerr = String.format("\nSTDERR:\n%s", exec.getStderr());
+        if (error) {
+            Logger.info(this, String.format("\nSTDOUT:\n%s", exec.getStdout()));
+            Logger.error(this, outputerr);
+        } else {
+            Logger.warn(this, outputerr);
+        }
+    }
+
```
Comment (olegmoz) [inline src/test/java/com/artipie/test/TestContainer.java:151]: @genryxy that looks like too complicated logic for output. I'd suggest to just print stdout and stderr after every command executed as `DEBUG` logging message and throw stderr content as exception if error code is not zero.

File: src/test/java/com/artipie/test/TestContainer.java [hunk 2]
```diff
@@ -88,7 +90,32 @@ public String execStdErr(final String... command) throws Exception {
...
+        if (code != 0) {
+            this.logStd(exec, true);
+            throw new IllegalStateException(
+                String.format(
+                    "'%s' failed with %s code", String.join(" ", command), code
+                )
+            );
+        }
+        this.logStd(exec, false);
+        return exec.getStdout().replace("\n", "");
```
Comment (olegmoz) [inline src/test/java/com/artipie/test/TestContainer.java:104]: @genryxy the code for replacing `\n` with empty string is duplicated 3 times in this class, I'd suggest to extract a function for that

**Review by olegmoz — CHANGES_REQUESTED** [review]
@genryxy thanks, please see two comments

**Comment by genryxy** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@genryxy thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#827` is not assigned, can't get performer

**Pull request merged**